asm/helper.go

```diff
@@ -458,6 +459,12 @@ func (gen *generator) irFuncAttribute(old ast.FuncAttribute) ir.FuncAttribute {
 		}
 	case *ast.FuncAttr:
 		return asmenum.FuncAttrFromString(old.Text())
+	case *ast.Preallocated:
+		ty, err := gen.irType(old.Typ())
```

Comment:
Use `typ` instead of `ty` for consistency with other variables names for LLVM types. For instance, the `typ` function parameter of [irInlineAsm](https://github.com/llir/llvm/blob/b3c562869eba32de2477858d77493dd19cc67419/asm/helper.go#L547).